README.md

```diff
@@ -100,6 +112,63 @@ Hit `Ctrl+C` to stop recording. If no audio device index (`-d`) is provided, the
 
 ### Go
 
+To install the PvRecorder Go module to your project, use the command:
+
+```console
+go get github.com/Picovoice/pvrecorder/binding/go
+```
+
+To get the list of available devices:
+
+```go
+import . "github.com/Picovoice/pvrecorder/binding/go"
+
+devices, err := GetAvailableDevices()
+if err != nil {
+    // error
+}
+```
+
+To start recording, initialize the instance and run start function:
+
+```go
+import . "github.com/Picovoice/pvrecorder/binding/go"
+
+recorder := PvRecorder{
+    FrameLength: 512,
+    DeviceIndex: -1, // Using -1 for index uses default audio input device.
+    BufferedFramesCount: 10,
+}
```

Comment:
I think we need a `NewPvRecorder` constructor that gives us an instance with default args. We do this with a bunch of our engines and it's a pattern in Go.